adbe/main.py

```diff
@@ -78,7 +78,7 @@
     adbe [options] app signature <app_name>
     adbe [options] app backup <app_name> [<backup_tar_file_path>]
     adbe [options] install <file_path>
-    adbe [options] uninstall <app_name>
+    adbe [options] uninstall [-0] <app_name>
```

Comment:
How about `--first-user` instead of `-0` or something similar? ADB uses `-0` but it is not intuitive at all.

Reply:
I thought of the ID 0, `--first-user` or `--privileged` it's also more intuitive

Reply:
`--first-user` looks better than `-0`